graylog2-server/src/main/java/org/graylog2/plugin/Message.java

```diff
@@ -192,6 +192,9 @@ public DateTime getTimestamp() {
 
         for (Map.Entry<String, Object> entry : fields.entrySet()) {
             final String key = entry.getKey();
+            if (key.equals(FIELD_ID)) {
```

Comment:
The `_id` field musn't be included as a message field if it's supplied during indexing, otherwise indexing fails.